packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```
@@ -173,9 +173,665 @@
+        "required": [
+          "pageSize"
+        ],
+        "properties": {
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          },
+          "firstTxTime": {
+            "type": "object"
+          },
+          "lastTxTime": {
+            "type": "object"
+          }
+        }
+      },
+      "GetAccountTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          },
+          "firstTxTime": {
+            "type": "object"
+          },
+          "lastTxTime": {
+            "type": "object"
+          },
+          "firstTxHeight": {
+            "type": "integer"
+          },
+          "lastTxHeight": {
+            "type": "integer"
+          }
+        }
+      },
+      "GetAccountAssetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Asset Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "assetId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountAssetsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Assets.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "firstAssetId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Detail.",
+        "type": "object",
+        "required": [
+          "pageSize",
+          "paginationKey",
+          "paginationWriter"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "writer": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
```

```suggestion
            "type": "integer"
```
https://iroha.readthedocs.io/en/develop/develop/api/queries.html#get-account-detail and rather: https://iroha.readthedocs.io/en/develop/develop/api/queries.html#result-pagination